core/tests/protractor_utils/ReleaseCoordinatorPage.js

```diff
@@ -26,6 +26,14 @@ var ReleaseCoordinatorPage = function() {
     '.protractor-test-unfinished-one-off-jobs-rows'));
   var unfinishedOffJobIDClassName = (
     '.protractor-test-unfinished-one-off-jobs-id');
+  var oneOfJobsStartBtnLocator = by.css(
+    '.protractor-test-one-off-jobs-start-btn');
+  var oneOffJobsStopBtnLocator = by.css(
+    '.protractor-test-one-off-jobs-stop-btn');
+  var unfinishedOneOffJobsIdElement = element.all(
+    by.css('.protractor-test-unfinished-one-off-jobs-id'));
```

Comment:
```suggestion
  var unfinishedOneOffJobsIdElements = element.all(
    by.css('.protractor-test-unfinished-one-off-jobs-id'));
```

since this can be many elements

Reply:
Fixed!